spec/mailers/password_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe PasswordMailer, type: :mailer do
+    before(:all) do
+        @user = create :user_with_all_attributes
+    end
+
+    describe "reset" do
+        let(:mail) { PasswordMailer.with(user: @user).reset }
+
+        it "renders the headers" do
+            expect(mail.subject).to eq("Reset")
+            expect(mail.to).to eq([@user.email])
+            expect(mail.from).to eq(["jacksonstark77@hotmail.com"])
```

Comment:
Why is this "from" "jacksonstark77@hotmail.com"? That looks like the value that was set as the default in the `ApplicationMailer`. 

When I see this, I think about you being tied to this one address. The `mail` interface seems pretty flexible, but then we specialize it with things like the "reset" method. This seems to be an approach that people take with Rails, seems fine to me. 

In a professional context you are likely having a lot of different emails. 

- support@domain.com
- no-reply@domain.com
- reset@domain.com
etc..


In this case I guess you would create different mailers, and then for each one define the default as the address to use. This might be a hassle to store as environment variables. You also don't want to be copying these things around between your files. 

You also probably want different emails based on dev, staging, prod.. Maybe in dev you don't send email, but just log the contents of the email instead. 

I don't have a concrete change here, just something to think about.